dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -0,0 +1,57 @@
+[supervisord]
+logfile_maxbytes=1MB
+logfile_backups=2
+nodaemon=true
+
+[program:start.sh]
+command=/usr/bin/start.sh
+priority=1
+autostart=true
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:rsyslogd]
+command=/usr/sbin/rsyslogd -n
+priority=2
+autostart=false
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+{# If our configuration has VLANs... #}
+{% if VLAN %}
+[group:isc-dhcp-relay]
```

Comment:
If there are VLANs but non of them contains a "dhcp_server" attribute, there will be a `[group: isc-dhcp-relay]` with no `programs` specified. Will that work?

Reply:
It does work, but it isn't the cleanest solution.

Reply:
Well, it required 8 lines of code and 2 lines of comments to implement the check. I think it's still fairly readable, but I wish it could be simpler.